### PR TITLE
Add calendar styling and onclick functionality to go to next and previous months

### PR DESCRIPTION
**Pull request by michelledhuang23**

### CODE REVIEW

File: client/dist/error/index.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
```
Comment (sbtorres) [inline client/dist/error/index.html]: What does this do? I've never seen it before.

File: client/src/index.jsx [hunk 2]
```diff
@@ -12,7 +12,12 @@ class BookingPortal extends React.Component {
   }
 
   componentDidMount() {
-    fetch('/booking?listingid=2', { method: 'GET' })
+    const params = new URLSearchParams(window.location.search);
```
Comment (sbtorres) [inline client/src/index.jsx]: Nice!

File: client/src/calendar.jsx [hunk 3]
```diff
@@ -1,79 +1,39 @@
...
-  text-align: center;
-  color: rgb(72, 72, 72);
-  width: 40px;
-  height: 40px;
-
-  &:hover {
-    background: rgb(228, 231, 231);
-  }
-`;
+import {
```
Comment (sbtorres) [inline client/src/calendar.jsx]: This is really clean, I'm going to have to use it too!

File: client/dist/error/index.html [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Error</title>
+</head>
+<body>
+  <h2>OOPS</h2>
+</body>
+</html>
```
Comment (dorktobes) [inline client/dist/error/index.html]: is this file necessary?

File: client/src/index.jsx [hunk 5]
```diff
@@ -12,7 +12,12 @@ class BookingPortal extends React.Component {
...
+    if (!params.has('listingid')) {
+      window.location.href = '/error';
```
Comment (dorktobes) [inline client/src/index.jsx]: ahh, this is why you have that error page above.

File: client/dist/error/index.html (hunk 1 above)
Comment (michelledhuang23) [reply client/dist/error/index.html]: oh I created a page to handle invalid listings ^^
